evaporate.js

```diff
@@ -432,10 +458,54 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
            authorizedSend(complete);
         }
 
+        var numProcessed = 0,
+            numParts = -1;
+
+        function computePartMd5Digest(part) {
+           return function () {
+              var s = me.file.status;
+              if (s == ERROR || s == CANCELED) {
+                 return;
```

Comment:
This is a fast abort to do nothing if the file is in Error or has been Canceled. As the computation "jobs" may be queued up, this just saves a lot of time and keeps us from doing things that are no longer needed.